dipy/reconst/dti.py

```diff
@@ -1856,17 +1953,23 @@ def restore_fit_tensor(design_matrix, data, sigma=None, jac=True,
             sigma = np.reshape(sigma, (-1, 1))
 
     # calculate OLS solution
-    D = ols_fit_tensor(design_matrix, flat_data, return_lower_triangular=True)
+    D, _ = ols_fit_tensor(design_matrix, flat_data, return_lower_triangular=True)
 
     # Flatten for the iteration over voxels:
     ols_params = np.reshape(D, (-1, D.shape[-1]))
 
     # Initialize parameter matrix
     params = np.empty((flat_data.shape[0], npa))
 
+    # For storing whether image is used in final fit for each voxel
+    robust = np.ones(flat_data.shape, dtype=int)
+
     # For warnings
     resort_to_OLS = False
 
+    # Instance of _nlls_class, need for nlls error func and jacobian
+    nlls = _nlls_class()
```

Comment:
Calling this object `nlls` here could be slightly confusing, because there is also a scipy optimization function of that name.

Reply:
that's true, but within the function scope, surely it's okay to use this name?

Reply:
I don't think there is a scipy opt functions with this name... there is one called `nnls` which is different. Leaving this for now